README.md

```diff
@@ -117,7 +146,7 @@ If you want to run all tests on pull request #1234 that has been
 opened by someone who is not in our white-list, run tests-trigger
 with `-f`:
```

Comment:
Can you please also fix the `-f` here?

Reply:
Argh, silly me -- fixed, thanks for spotting!